estate_account/models/estate_property.py

```diff
@@ -0,0 +1,32 @@
+from odoo import models, api, fields
+class EstateProperty(models.Model):
+    _inherit = "estate.property"
+
+    #------------------------------------------Action Methods----------------------------------------------
+    def property_sold_action(self):
+        """set property state as sold and Create invoice for sold property"""
+        res = super().property_sold_action()
+        journal = self.env["account.journal"].search([("type", "=", "sale")], limit=1)
```

Comment:
```suggestion
        journal = self.env["account.journal"].search([("type", "=", "sale")], limit=1)
```
why do you search for a sale journal?
What do you understand by journal?

Reply:
sir, we need a journal to be known by odoo to store account transactions and since we are generating invoice for a sales order, we look for a sales journal.